src/pivot-table/components/DimensionCell.tsx

```diff
@@ -65,23 +72,22 @@ const DimensionCell = ({
   constraints
 }: DimensionCellProps): JSX.Element => {
   const { qText, qCanCollapse, qCanExpand } = (cell.value as NxPivotDimensionCell);
-  let cellContent = qText;
   let onClickHandler: (() => void) | undefined;
+  let cellIcon = null;
 
   if (qCanExpand) {
-    cellContent = `[ + ] ${qText}`;
+    cellIcon = <AddCircleOutlineSharpIcon fontSize="small" />
     onClickHandler = createOnExpand({ model, isLeftColumn, rowIndex, colIndex, constraints });
   } else if (qCanCollapse) {
-    cellContent = `[ - ] ${qText}`;
+    cellIcon = <RemoveCircleOutlineSharpIcon fontSize="small" />
     onClickHandler = createOnCollapse({ model, isLeftColumn, rowIndex, colIndex, constraints });
   }
 
   return (
     <div style={{ ...style, ...containerStyle}}>
       <div style={{ ...cellStyle, ...borderStyle }} onClick={onClickHandler} aria-hidden="true">
-        <div style={dimTextStyle}>
-            {cellContent}
-        </div>
+        {cellIcon}
+        <div style={dimTextStyle}>{qText}</div>
```

Comment:
You should check out mui `styled` (not the same as styled components). Similar to `makeStyles` but supported by mui5. Then you don't have to do inline styling, which i think is slower. Some good examples in here: https://github.com/qlik-oss/sn-table/pull/426/files

Reply:
Not relevant to this PR though

Reply:
Agree, I will, at some point, have to take a look at styling. Including theming. But for now this will have to do.